runtime/sql/sql.go

```diff
@@ -64,6 +66,35 @@ func (i *Isolate) Close() error {
 	return i.library.Close()
 }
 
+func (i *Isolate) request(request *requests.Request) *requests.Response {
+	f, err := i.library.FindFunc("request")
+	if err != nil {
+		panic(err)
+	}
+
+	thread := i.attachThread()
+
+	bytes, _ := proto.Marshal(request)
```

Comment:
Best to handle the returned error and return it. Just change the function signature to:

```go
func (i *Isolate) Request(req *requests.Request) (*requests.Response, error) {
```